server/sonar-web/src/main/js/apps/component-measures/drilldown/CodeView.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import key from 'keymaster';
+import SourceViewer from '../../../components/SourceViewer/SourceViewer';
+import { isDiffMetric } from '../../../helpers/measures';
+import { parseDate } from '../../../helpers/dates';
+/*:: import type { ComponentEnhanced, Paging, Period } from '../types'; */
+/*:: import type { Metric } from '../../../store/metrics/actions'; */
+
+/*:: type Props = {|
+  branch?: string,
+  component: ComponentEnhanced,
+  components: Array<ComponentEnhanced>,
+  leakPeriod?: Period,
+  metric: Metric,
+  selectedIdx: ?number,
+  updateSelected: string => void,
+|}; */
+
+export default class CodeView extends React.PureComponent {
+  /*:: props: Props; */
+
+  componentDidMount() {
+    this.attachShortcuts();
+  }
+
+  componentWillUnmount() {
+    this.detachShortcuts();
+  }
+
+  attachShortcuts() {
+    key('j', 'measures-files', () => {
```

Comment:
you will be surprised, but "j" means next and "k" means previous 😉

Reply:
Ah yeah true, down should be next and up should be back, thanks